language/e2e-tests/src/account.rs

```diff
@@ -864,14 +761,13 @@ impl AccountData {
     }
 
     /// Creates and returns the top-level resources to be published under the account
-    pub fn to_value(&self) -> (Value, Vec<(Identifier, Value)>, Value, Value) {
+    pub fn to_value(&self) -> (Value, Vec<(Identifier, Value)>, Value) {
```

Comment:
That is the reason for the other function. Even before this PR, the E2E account type creaton wasn't working (has been broken since the big signer refactor)... We need to fix it in the near future, but I would like to do that in the "right" way (invoke the appropriate creation function from the fake executor rather than hardcoding the layout of each account in Rust...)

Reply:
Sounds good! I agree that we want to unbork this the "right" way, and not spend too much time trying to create a middle-ground solution. 

I can't possibly imagine why hardcoding the layout of each account in Rust would be painful at all.....